galactic/release-build.yaml

```diff
@@ -0,0 +1,94 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+
+build_environment_variables:
+  RTI_NC_LICENSE_ACCEPTED: 'yes'
+
+jenkins_binary_job_priority: 86
+jenkins_binary_job_timeout: 120
+jenkins_source_job_priority: 70
+jenkins_source_job_timeout: 30
+notifications:
+  emails:
+  - scott+build.ros2.org@openrobotics.org
+  - ros2-buildfarm-galactic@googlegroups.com
+  maintainers: true
+sync:
+  package_count: 262
```

Comment:
Do we want to lower the package sync count temporarily, or just assume that we'll have enough packages in Galactic in short-order that this will work?

Reply:
There are enough vendor packages in rolling to fill in the gaps, and I'm afraid that anything less than this would mean that something in the core is broken. We can lower it later, but the optimistic expectation is that everything building in rolling will build successfully in galactic.